server/services/user.js

```diff
@@ -0,0 +1,172 @@
+//  Defines various services for the user object
+
+const Sequelize = require('sequelize');
+const sequelize = require('../../settings/connect');
+const User = require('../../app/models/user')(sequelize, Sequelize);
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt-nodejs');
+
+const secret = 'documentmanagement'; // Specify a secret to sign json web tokens
```

Comment:
There are other parts in the code with `secret = process.env.SECRET || 'documentmanagement'`. This will lead to conflict whenever the user decides to declare SECRET as part of the environment variables